Essentials/src/config.yml

```diff
@@ -371,6 +371,8 @@ unprotected-sign-names:
 backup:
   # Interval in minutes.
   interval: 30
+  # If true, the backup task will run even if there are no players online.
+  always-run-backup: false
```

Comment:
I suggest changing this from `always-run-backup` to simply `always-run` to avoid redundancy since this config option is already a child of `backup`.